### PR TITLE
feat: handle title bill

### PR DESCRIPTION
**Pull request by DomeT99**
Hello!
I open this PR where I implemented a dynamic management of the title of each bill :)

### CODE REVIEW

File: src/routes/index.tsx [hunk 1]
```diff
@@ -34,6 +35,15 @@ export default component$(() => {
       <main>
         <div class="flex flex-col gap-10 items-center">
           <h2 class="text-3xl">Split your travel expenses easily</h2>
+          <div>
+            <input
+              type="text"
+              class="w-full shadow border-2 rounded p-1"
+              placeholder="Bill name"
+              value={titleBill.value}
+              onInput$={(_, el) => (titleBill.value = el.value)}
```
Comment (Balastrong) [inline src/routes/index.tsx]: You can replace it with the shorter syntax, what do you think?

```suggestion
              bind:value={titleBill}
```

**Review by Balastrong — CHANGES_REQUESTED** [review]
Thanks for the PR! I left a comment :)

File: src/routes/index.tsx (hunk 1 above)
Comment (DomeT99) [reply src/routes/index.tsx]: Sure, it's more clean

**Review by Balastrong — APPROVED** [review]
LGTM, thank you!

**Comment by Balastrong** [comment]
@all-contributors please add @DomeT99 for code

**Pull request merged**